src/test/java/spoon/test/jar/JarTest.java

```diff
@@ -27,7 +27,7 @@ public void testJar() throws Exception {
 				SpoonResourceHelper.resources("./src/test/resources/sourceJar/test.jar"));
 		Assert.assertTrue(compiler.build());
 		assertEquals(1, factory.getModel().getAllTypes().size());
-		assertEquals("spoon.test.strings.Main", factory.getModel().getAllTypes().iterator().next().getQualifiedName());
+		assertEquals("spoon.test.strings.testclasses.Main", factory.getModel().getAllTypes().iterator().next().getQualifiedName());
```

Comment:
I don't agree with changing this equals and the related jar file: the purpose of moving the resource file is to ease the maintaining by avoid polluting test classes with resources at the same place. But here the file is already in a jar file, so it does not pollute anything and changing the package name does not change anything to the test.